spynnaker/pyNN/spynnaker.cfg

```diff
@@ -30,7 +30,7 @@ one_to_one_connection_dtcm_max_bytes = 2048
 # MallocBasedRoutingInfoAllocator, GraphEdgeFilter, EdgeToNKeysMapper
 
 application_to_machine_graph_algorithms = PartitionAndPlacePartitioner
-machine_graph_to_machine_algorithms = GraphEdgeFilter,OneToOnePlacer,NerRoute,BasicTagAllocator,EdgeToNKeysMapper,ProcessPartitionConstraints,MallocBasedRoutingInfoAllocator,BasicRoutingTableGenerator
+machine_graph_to_machine_algorithms = GraphEdgeFilter,EdgeToNKeysMapper,OneToOnePlacer,NerRoute,BasicTagAllocator,EdgeToNKeysMapper,ProcessPartitionConstraints,MallocBasedRoutingInfoAllocator,BasicRoutingTableGenerator
```

Comment:
This *will* run EdgeToNKeysMapper twice...

Reply:
yes that is a mistake. Will fix.